WordPress/Classes/ViewRelated/Gutenberg/Layout Picker/GutenbergLayoutPickerViewController.swift

```diff
@@ -0,0 +1,193 @@
+import UIKit
+import Gridicons
+
+class GutenbergLayoutPickerViewController: UIViewController {
+
+    @IBOutlet weak var headerView: UIView!
+    @IBOutlet weak var closeButton: UIButton!
+    @IBOutlet weak var titleView: UILabel!
```

Comment:
Specifically using a `titleView` here instead of the navigation bar's title because there was a weird random animation issue when scrolling quickly and setting the title on the screen the label would "fly" in from the upper left.